apps/supervisor/.env.example

```
@@ -14,4 +14,7 @@ OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:3030/otel
 
 # Optional settings
 DEBUG=1
-TRIGGER_DEQUEUE_INTERVAL_MS=1000
\ No newline at end of file
+TRIGGER_DEQUEUE_INTERVAL_MS=1000
+
+# Dev-only: stream this process's logs over a local telnet/TCP socket (nc localhost 6769). Unset to disable.
+SUPERVISOR_TELNET_LOGS_PORT=6769
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)